fastlane/bin/fastlane

```diff
@@ -1,5 +1,17 @@
 #!/usr/bin/env ruby
 $LOAD_PATH.push File.expand_path("../../lib", __FILE__)
 
+unless ENV['BUNDLE_BIN_PATH']
+  has_gemfile = File.exist?('Gemfile')
+  File.open('Gemfile', 'w') { |file| file.write("gem 'fastlane'") } unless has_gemfile
```

Comment:
Could we also create this file in a temporary directory, and use it from there? This way, users won't see the Gemfile when they look at their git client while fastlane is running